pedra_papel_tesoura/spec/pedra_papel_tesoura_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_base'
+require 'pedra_papel_tesoura'
+
+describe 'Pedra, Papel e Tesoura' do
+  it 'deve retornar o cenário de derrota do jogador quando usa pedra contra papel' do
+    expect(jogar('pedra', 'papel')).to eq('Você perdeu - pedra perde contra papel')
+  end
+
+  it 'deve retornar o cenário de derrota do jogador quando usa papel contra tesoura' do
+    expect(jogar('papel', 'tesoura')).to eq('Você perdeu - papel perde contra tesoura')
+  end
+
+  it 'deve retornar o cenário de derrota do jogador quando usa tesoura contra pedra' do
+    expect(jogar('tesoura', 'pedra')).to eq('Você perdeu - tesoura perde contra pedra')
+  end
+
+  it 'deve retornar quando ambos utilizam pedra' do
```

Comment:
Acho que faltou aqui, ne?
```suggestion
  it 'deve retornar o cenário de empate, quando ambos utilizam pedra' do
```